### PR TITLE
docs - update guc gp_enable_relsize_collection - ORCA support

### PR DESCRIPTION
**Pull request by mkiyama**
Will be ported to 5X_STABLE.

See dev PR https://github.com/greenplum-db/gpdb/pull/5322

### CODE REVIEW

**Comment by mkiyama** [comment]
@oarap @vraghavan78 
Could you give this a quick review?

thanks

**Comment by mkiyama** [comment]
@oarap @vraghavan78
Could you give this a quick review?

thanks

File: gpdb-doc/dita/admin_guide/query/topics/query-piv-opt-notes.xml [hunk 1]
```diff
@@ -43,6 +43,11 @@
       <li><codeph>optimizer_sort_factor</codeph> controls the cost factor that GPORCA applies to
         sorting operations during query optimization. The cost factor can be adjusted for queries
         when data skew is present. </li>
+      <li><codeph>gp_enable_relsize_collection</codeph> controls how GPORCA (and the legacy query
+        optimizer) handle a table without statistics. By default, GPORCA uses a default value to
+        estimate the number of rows if statistics are not available. When this value is
+          <codeph>on</codeph>, GPORCA uses the estimated size of a table if there are no statistics
+        for the table.</li>
```
Comment (vraghavan78) [inline gpdb-doc/dita/admin_guide/query/topics/query-piv-opt-notes.xml]: gp_enable_relsize_collection will not be triggered for root partition statistics. Not sure if that is worth mentioning.

Comment (mkiyama) [reply gpdb-doc/dita/admin_guide/query/topics/query-piv-opt-notes.xml]: @vraghavan78 
So, if the root of a partitioned table doesn't have statistics, the default value is used?

For ORCA you should use ANALYZE ROOT PARTITION to ensure a the root of a partitioned table has statistics?

or something like that?

thanks

**Pull request merged**